app/main.py

```diff
@@ -1,3 +1,20 @@
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @classmethod
+    def from_dict(cls, course_dict):
+        name = course_dict["name"]
+        description = course_dict["description"]
+        weeks = OnlineCourse.days_to_weeks(course_dict["days"])
+        return OnlineCourse(name, description, weeks)
```

Comment:
```suggestion
        return cls(name, description, weeks)
```
It's better to use `cls` reference to the class than "hardcode" its name. The difference will be discussed on the OOP topics.